src/applications/ivc-champva/10-7959C/tests/10-7959C.cypress.spec.js

```diff
@@ -128,9 +124,13 @@ const testConfig = createTestConfig(
               // format of dates (goes from YYYY-MM-DD to MM-DD-YYYY).
               // TODO: Address discrepancy at some point.
               expect(data[k]?.length).to.equal(req.body[k]?.length);
+            } else {
+              expect(data[k] === req.body[k]);
```

Comment:
Base case where we just check that the data input into the form matches what we get on the other side.